### PR TITLE
Move location data to a .json file

### PR DESCRIPTION
**Pull request by oxytocinlove**
Data should not be in code files. This also makes it easier to do partial updates and avoids the need for custom formatting when writing the data.

### CODE REVIEW

**Comment by beshaya** [comment]
It's pretty hard to review this with all the csv files in the same commit as the code changes. Any chance you could remove the csv files from this pull request or rebase them into a separate commit?

**Comment by oxytocinlove** [comment]
@beshaya done!

**Comment by oxytocinlove** [comment]
(Before merging I'll run the script again to ensure it's not merging in old data)

File: src/data/locationHelpers.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { Locations, PrevalenceDataDate } from 'data/location'
+import { Locations } from 'data/location'
```
Comment (beshaya) [inline src/data/locationHelpers.ts:1]: Will importing the json file like this check that types are correct? If not, is there loading we can do to make sure that the data in the file actually matches the PrevalenceData interface?

**Pull request closed without merging**